README.md

```
@@ -1,3 +1,5 @@
+Sneaking in a random change, wondering if someone notices.
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)